core/new-gui/src/app/workspace/component/operator-panel/operator-label/operator-label.component.spec.ts

```diff
@@ -67,6 +69,22 @@ describe('OperatorLabelComponent', () => {
     expect(jqueryElement.data('uiDraggable')).toBeTruthy();
   });
 
+  it('should display operator description shortly after hovering on a operator label', () => {
+    const spy = spyOn<any>(component, 'mouseEnter');
+    const operatorLabelElement = fixture.debugElement.query(By.css('#' + component.operatorLabelID));
+    operatorLabelElement.triggerEventHandler('mouseenter', component);
+    expect(spy).toHaveBeenCalled();
+    expect(mockOperatorData.additionalMetadata.operatorDescription).toContain(operatorLabelElement.attributes['ng-reflect-ngb-tooltip']);
```

Comment:
We should test the `mouseEnter` event at `operatorLabelElement` rather than at `component`. Also, use `marble` to test if the given event is triggered at given time.